### ISSUE TITLE
Private Channel Errors

### ISSUE THREAD

**Issue by GeekyDeaks** [GeekyDeaks/discord-destinybot#2]
Some of the commands rely on information that is only available on a server channel.  This should be cleanly dealt with instead of providing a obscure error message.

**Comment by GeekyDeaks** [comment]
commit b3f64e5 sets a default server, so that when commands are issued via PM, it can fall back to a server and still work.  This is fine for 1:1 bot:server setups, which since there is bugger all per-server logic anywhere to be seen, should be fine for now.  

This also allows users to decide if verbose commands should be kept private or flood the server channel.

**Comment by GeekyDeaks** [comment]
Phillip, let me know if this fix addresses your wish to have the bot PM responses.  If we need multiple server support then it's not going to work...

**Comment by unisys12** [comment]
Will do. Hopefully my internet will be working when I get home from work
this afternoon.

**Comment by GeekyDeaks** [comment]
Added further logic to allow responses to be public /private in commit 2ebda29.  By default all messages are private unless the keyword `public` is at the end of the command.  Also by default, all public errors will be deleted after 10 seconds.

To achieve this I abstracted out the send and update messages to a separate module `bot/messages.js`.  The functions in there also take into account the maximum message size and break up the text into smaller messages.  This **only** works if you send an array though, which by default will be joined with `\n`
